### PR TITLE
Not wrap header elements

### PR DESCRIPTION
**Pull request by McGiverGim**
When we reduce at minimum resolution the Configurator, and the Virtual FC is enabled, the elements in the header go wrong:
image

This fixes that, the logo is reduced to the minimum, but there is not space to play more:
image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
Should we also hide `firmware-virtual-options` if connected?

image

**Comment by haslinghuis** [comment]
Don't see any changes switching firmware versions.

**Review by haslinghuis — APPROVED** [review]

**Comment by McGiverGim** [comment]
@haslinghuis I think this is the place where it is shown or hide:
https://github.com/betaflight/betaflight-configurator/blob/d92a49aac8a1c079e812cb7e361c4383a460010b/src/js/serial_backend.js#L8-L28

The strangest think is that I see this continuosly being called, while connected too. Maybe the `port-picker` is continuosly being updated? Then something is wrong with it.
https://github.com/betaflight/betaflight-configurator/blob/d92a49aac8a1c079e812cb7e361c4383a460010b/src/js/serial_backend.js#L40-L42

I think we have two options about the MSP combo. Mark it as read-only and let as information in the screen, or hide when connecting. But this two changes are out of the scope of this PR, we can try to do it in a future PR and look too to the reason why it is being refreshed continuosly.

**Comment by haslinghuis** [comment]
I think have fixed the issue with the port picker element change handler.
Please see: #2932

**Review by limonspb — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
